### PR TITLE
sources adaptives

### PR DESCRIPTION
**Pull request by Chynchenko**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss [hunk 1]
```diff
@@ -95,4 +95,73 @@ $articlesImg: "@assets/images/sources/Articles.png";
...
+  
+    &:hover {
+      text-decoration: underline;
+      cursor: pointer;
+    }
+  }
+  }
+  
+  .ant-modal-close {
+    @include mut.positioned-as(absolute, $left: 90%, $top: -15px);
```
Comment (Katerix) [inline src/app/common/components/modals/Sources/SourcesModal.styles.scss:156]: overriding lib classes like .ant-modal-close like this you ovverride all this components in project. Don't do it again please.

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss [hunk 2]
```diff
@@ -95,4 +95,73 @@ $articlesImg: "@assets/images/sources/Articles.png";
...
+    @include mut.circular(40px, c.$pure-white-color); 
+    &:hover {
+      background-color: c.$modal-hover-color;
+    }
+    .ant-modal-close-x {
+      display: flex;
```
Comment (Katerix) [inline src/app/common/components/modals/Sources/SourcesModal.styles.scss:162]: same

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss [hunk 3]
```diff
@@ -95,4 +95,73 @@ $articlesImg: "@assets/images/sources/Articles.png";
...
+  .mainContentContainer {
+  width: 340px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  }
+  
+  .sourcesModal .ant-modal-content {
+  height: 370px;
```
Comment (Katerix) [inline src/app/common/components/modals/Sources/SourcesModal.styles.scss:116]: remove space between classes on line 115

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss [hunk 4]
```diff
@@ -95,4 +95,73 @@ $articlesImg: "@assets/images/sources/Articles.png";
...
+
+@media (max-width: 480px) {
+
+  .sourcesModal {
+  width: 340px;
+  height: 442px;
+  }
+  
+  .mainContentContainer {
+  width: 340px;
```
Comment (Katerix) [inline src/app/common/components/modals/Sources/SourcesModal.styles.scss:108]: it is a better practice to use more specific selecctors

**Review by Katerix — CHANGES_REQUESTED** [review]

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss (hunk 1 above)
Comment (Chynchenko) [reply src/app/common/components/modals/Sources/SourcesModal.styles.scss:156]: I'll take it

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss (hunk 2 above)
Comment (Chynchenko) [reply src/app/common/components/modals/Sources/SourcesModal.styles.scss:162]: +

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss (hunk 3 above)
Comment (Chynchenko) [reply src/app/common/components/modals/Sources/SourcesModal.styles.scss:116]: ok)

File: src/app/common/components/modals/Sources/SourcesModal.styles.scss (hunk 4 above)
Comment (Chynchenko) [reply src/app/common/components/modals/Sources/SourcesModal.styles.scss:108]: ok)

**Pull request merged**